src/main/java/org/apache/flink/state/benchmark/RescalingBenchmarkBase.java

```diff
@@ -57,6 +63,26 @@ public static void runBenchmark(Class<?> clazz) throws RunnerException {
         new Runner(options).run();
     }
 
+    protected static File prepareDirectory(String prefix) throws IOException {
```

Comment:
All code paths is similar with StateBenchmarkBase#createKeyedStateBackend and StateBackendBenchmarkUtils#prepareDirectory, Could we merge them together ?

Reply:
Hi @masteryhx ,  The `StateBackendBenchmarkUtils` belongs to the Flink repo and the `prepareDirectory` is a private method. I'd rather not change the Flink repo for the benchmark needs, WDYT?

Reply:
Fine. I not insisted on it.